articles/architecture-scenarios/application/spa-api.md

```diff
@@ -2,27 +2,31 @@
 order: 01
 title: SPA + API
 image: /media/articles/architecture-scenarios/spa-api.png
-extract: Single Page Web Application “Client” which talks to an API (“Resource Server”). The application will use OpenID Connect with the Implicit Grant Flow to authenticate users with Auth0.
+extract: Single Page Web Application "Client" which talks to an API ("Resource Server"). The application will use OpenID Connect with the Implicit Grant Flow to authenticate users with Auth0.
 description: Explains the architecture scenario with a Single Page Web Application to an API using OpenID Connect with the Implicit Grant Flow to authenticate users with Auth0.
 ---
 
 # SPA + API
 
-![](/media/articles/architecture-scenarios/spa-api.png)
+![SPA + API Flow](/media/articles/architecture-scenarios/spa-api.png)
 
-In this scenario you have a Single Page Web Application “Client” which talks to an API (“Resource Server”). The application will use OpenID Connect with the Implicit Grant Flow to authenticate users with Auth0.
+In this scenario you have a Single Page Web Application ("Client") which talks to an API ("Resource Server"). The application will use **OpenID Connect** with the **Implicit Grant Flow** to authenticate users with Auth0. Note that this flow can only be used for Clients whose type is **Single Page Application** in the [Dashboard](${manage_url}).
 
-When a user logs in, Auth0 will return to the application an `access_token` as well as an `id_token`. The `id_token` is used to securely call the API on behalf of the user. Alternatively the user profile can be obtained by calling the `/userinfo` endpoint in the Auth0 Authentication API with the `access_token`.
+When a user logs in, Auth0 will return to the application an `access_token` and optionally an `id_token`:
```

Comment:
This is the other way around - an ID token will always be returned for the client. An access token always will be returned unless the response type was `id_token`: https://openid.net/specs/openid-connect-core-1_0.html#ImplicitAuthResponse